docs/src/man/hosting.md

```diff
@@ -128,9 +128,11 @@ jobs:
       julia: 1.0
       os: linux
       script:
-        - julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd()));
-                                               Pkg.instantiate()'
-        - julia --project=docs/ docs/make.jl
+         - julia -e 'using Pkg; Pkg.activate(); push!(LOAD_PATH, pwd());' 
+        - julia --project=docs/ -e 'using Pkg; Pkg.activate();  push!(LOAD_PATH,pwd());'
```

Comment:
These two first lines does not do anything since they are undone the second Julia quits, and you shouldn't have to modify the LOAD_PATH anyway.

Reply:
Travis gave me errors in the documentation stage when I used the original config:
https://travis-ci.com/juliamatlab/MatLang/jobs/232115453/config

But I fixed the issue by adding these two lines:
https://travis-ci.com/juliamatlab/MatLang/jobs/232117413/config

I think it was because my package is not registered. 

I got those errors when building the package offline in Windows using PowerShell.

Reply:
Hmm. I am wondering if this is a bug in nightly somehow. @fredrikekre Do you see why the `julia --project=docs/ docs/make.jl` call did not detect the docs project/manifest in the first log?

The `LOAD_PATH`s should definitely not be necessary though.

Reply:
I didn't upload `manifest` for the docs folder just a `project.toml`. So I thought maybe I need to write the script manually to create that.

Reply:
You seem to have deactivated the docs environment you start julia with: https://travis-ci.com/juliamatlab/MatLang/jobs/232115453#L225

Reply:
Indeed: https://github.com/juliamatlab/MatLang/blob/3b287f7db3cc2ef115441d1498594ab10d8342db/docs/make.jl#L9

Reply:
https://github.com/juliamatlab/MatLang/blob/master/docs/make.jl
I don't think so. In a commit before these two, I had some script but I removed it.
These two are for the same `make.jl` file. I forced pushed my commit to remove history, however.

Reply:
The commit for the original Travis job definitely had the `make.jl` stuff (https://github.com/juliamatlab/MatLang/blob/3b287f7db3cc2ef115441d1498594ab10d8342db/docs/make.jl#L7-L10) and that is what caused the problem, not anything in `.travis.yml`.

Reply:
Let me see if I can recreate the error with the latest `make.jl` file but the old Travis code

Reply:
So the original Travis file works. It was my fault that I thought I removed that code from the `make.jl` file! 👀 
https://travis-ci.com/juliamatlab/MatLang/jobs/232190010